ramp-database/ramp_database/tools/tests/test_submission.py

```diff
@@ -506,3 +510,8 @@ def test_add_submission_similarity(session_scope_module):
     assert similarity.target_submission == target_submission
     assert similarity.similarity == pytest.approx(0.5)
     assert isinstance(similarity.timestamp, datetime.datetime)
+
+
+def test_compute_contributivity(session_scope_module):
+    compute_contributivity(session_scope_module, 'iris_test')
+    compute_historical_contributivity(session_scope_module, 'iris_test')
```

Comment:
This is not a test. You need to check the value which are added in the database to be sure that there is no bug in the process to compute the contributivity. This is too brittle now.

Reply:
I made an attempt but I'll give up for now. We'd need to train a second submission in the test for historical contributivity to be tested, and I'm not sure how I should do that. Blindly copying stuff from functions above didn't work.

Reply:
Compute_contributivity has assertions now, on the single trained submission.